### PR TITLE
WDS updates

### PR DESCRIPTION
**Pull request by crpietschmann**
June 2023 WDS updates

### CODE REVIEW

File: Whiteboard design session/WDS trainer guide - Building the business migration case with Windows Server and SQL Server.md [hunk 1]
```diff
@@ -220,6 +222,8 @@ The first workload they want to migrate to Azure consists of:
     - Database 2: 435 GB
     - Database 3: 50 GB
 
+This initial workload is not a mission critical application. However, it is representative of the typical VMs, databases and application architectures that are common across many of the other workloads that will be migrated to Azure. They've identified this workload as a good candidate for the first phase, POC migration to gain their confidence in migrating their mission critical workloads in later phases of the migration migration.
```
Comment (DawnmarieDesJardins) [inline Whiteboard design session/WDS trainer guide - Building the business migration case with Windows Server and SQL Server.md:225]: Last sentence has migration migration.

Comment (DawnmarieDesJardins) [reply Whiteboard design session/WDS trainer guide - Building the business migration case with Windows Server and SQL Server.md:225]: Sorry, for some reason, I don't have the ability to edit your PR so I can't fix myself.

In the Objections section...#4 answer - Azure Arc provides the ability manage all your....  Need to add **the** ability to manage...

thanks

File: Whiteboard design session/WDS student guide - Building the business migration case with Windows Server and SQL Server.md [hunk 2]
```diff
@@ -69,20 +69,22 @@ Directions: With all participants in the session, the facilitator/SME presents a
 
 Tailspin Toys is a global manufacturer of children’s toys that was founded in 1957 with its global headquarters located in Milwaukee, WI. Their mission-critical workloads are currently hosted in an on-premises datacenter and they are beginning a journey to modernize and migrate into the cloud using Microsoft Azure.
```
Comment (DawnmarieDesJardins) [inline Whiteboard design session/WDS student guide - Building the business migration case with Windows Server and SQL Server.md:70]: Please add a comma - ... hosted in an on-premises datacenter**,** and they are......

**Comment by DawnmarieDesJardins** [comment]
@crpietschmann 
Thanks Chris!  I only marked one of the documents....you'll need to update all 3 where needed.  Sorry, they're so minor, I'd have done it myself, but the edit button wasn't available for me.  Let me know if you have any questions.
Dawnmarie

**Comment by crpietschmann** [comment]
@DawnmarieDesJardins No problem! I fixed these items.

**Comment by DawnmarieDesJardins** [comment]
Thanks!  Quick double check....did you make the same changes to the PPT?

**Comment by crpietschmann** [comment]
> Thanks! Quick double check....did you make the same changes to the PPT?

Yes I updated the PPT too.

**Pull request merged**